pkgs/development/libraries/antic/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, mpir
+, gmp
+, mpfr
+, flint
+}:
+
+stdenv.mkDerivation rec {
+  pname = "antic";
```

Comment:
Is this still relevant since you opened an other PR for adding antic?

Reply:
This branch was just forked off the other branch, since it depends on antic. But once that goes in, I can rebase it on master.